src/sgame/sg_struct.h

```diff
@@ -772,9 +772,9 @@ struct level_locals_s
 
 	struct {
 		int synchronous;
-		int fixed;
```

Comment:
I changed the cvar the value is taken from to have `bool` type, and then changed this to prevent some warnings about assigning int to bool or vice versa.